src/sensor_imu_plugin.cpp

```diff
@@ -56,6 +54,40 @@ SensorImuPlugin::SensorImuPlugin(JSBSim::FGFDMExec* jsbsim)
 
 SensorImuPlugin::~SensorImuPlugin() {}
 
+void SensorImuPlugin::setSensorConfigs(TiXmlElement* configs) {
+  if (CheckConfigElement(configs, "gyroscope_noise_density")) {
+    GetConfigElement<double>(configs, "gyroscope_noise_density", gyroscope_noise_density);
+  }
```

Comment:
Keep this as simple as you can.
Why not just (and maybe pass a default value) (and the template argument should be inferred automatically):
```suggestion
  GetConfigElement(configs, "gyroscope_noise_density", gyroscope_noise_density[, 0.]);
```

Reply:
Did you see this? ^^
This should only be 1 line of code instead of 3.